### PR TITLE
added readable move description to map log output

### PR DESCRIPTION
**Pull request by freecraver**
simple change for better log output - unsure about how card actions should be logged. 
Note: merge base is currently set to temp/eval

### CODE REVIEW

File: src/main/java/util/game/TransparentRisk.java [hunk 1]
```diff
@@ -111,4 +115,35 @@ private String createContinentRateOutput(Map<Integer, Double> occupancyRateMap)
...
+        if (board.isReinforcementPhase()) {
+            return String.format("Reinforce %s (%d)", TerritoryNames.of(action.defendingId()), action.troops());
+        } else if (board.isAttackPhase()) {
+            if (action.attackingId() == -1) {
+                return String.format("Attack lost %d, Defence lost %d", action.attackerCasualties(), action.defenderCasualties());
+            }
+            return String.format("Attack %s -> %s (%d)", TerritoryNames.of(action.attackingId()),
+                    TerritoryNames.of(action.defendingId()), action.troops());
+        } else if (board.isOccupyPhase()) {
+            return String.format("Occupying with %d", action.troops());
```
Comment (AntonOellerer) [inline src/main/java/util/game/TransparentRisk.java:142]: Could you maybe add the country being fortified here as well?

**Review by AntonOellerer — APPROVED** [review]
Very nice!
One thing: I recommend using the `RiskAction` object methods to determine what an action actually does, since this does all the relevant checks itself (e.g. `RiskAction::isCardIds()`)

File: src/main/java/util/game/TransparentRisk.java (hunk 1 above)
Comment (freecraver) [reply src/main/java/util/game/TransparentRisk.java:142]: occupy != fortify != reinforcement
The occupy action has no reference to the country afaik and always depends on the previous action (the successfull attack)

**Comment by freecraver** [comment]
> 
> 
> Very nice!
> One thing: I recommend using the `RiskAction` object methods to determine what an action actually does, since this does all the relevant checks itself (e.g. `RiskAction::isCardIds()`)

ah thanks I didn't see those. I started with attack, fortify, etc. and there we don't have any such methods. Weird class design indeed 🙃 But I'll fix it, when I touch the class again

**Pull request merged**